Editor/Gui/UiModel.cs

```diff
@@ -242,34 +244,41 @@ private void Init()
         }
         
         public static Guid HomeSymbolId = Guid.Parse("dab61a12-9996-401e-9aa6-328dd6292beb");
-
-        public override void Load()
+        
+        public override void Load(bool log)
         {
             // first load core data
-            base.Load();
-
-            var uiJson = new SymbolUiJson();
+            base.Load(log);
 
             var symbolUiFiles = Directory.GetFiles(OperatorTypesFolder, $"*{SymbolUiExtension}", SearchOption.AllDirectories);
-
-            foreach (var symbolUiFile in symbolUiFiles)
+            var symbolUiJsons = symbolUiFiles.AsParallel()

```

Comment:
I'm not a huge fan either but proper spacing allows things to be readable. I mainly used it because it made "smart" parallelization a breeze

the GC hit happens if you abuse them in hot paths instead of properly iterating. since this is happening on startup, I'm not worried.

though I could name a place function or two where i certainly ignored that advice 😅

Reply:
I solved this mainly by creating a local function